src/mixins/createFormFieldMixin.ts

```diff
@@ -116,9 +116,8 @@ const createFormMixin: FormMixinFactory = compose({
 				if (this.state && this.state.name) {
 					props.name = this.state.name;
 				}
-				if (this.state.disabled) {
-					props['disabled'] = 'disabled';
-				}
+
+				props['disabled'] = !!this.state.disabled;
```

Comment:
Since these are now typed, you can do `props.disabled` which is better.  Also why not cast as `Boolean(this.state.disabled)` to be more explicit?